pages/Getting Started/Installation.md

```diff
@@ -183,7 +186,7 @@ for more info refer to the
 
 Please note that since Ubuntu is generally behind with dependencies, it's not
 guaranteed that the build process will work at all. Even if it is, it's likely
-that it will break at some point in the future.
+that it will break at some point in the future.  **Always use the latest version of Ubuntu for the most up to date dependencies if building or installing.** Note: Your mileage may also differ as GDM does have sone bugs with Hypr check the master tutorial for more info.
```

Comment:
```suggestion
that it will break at some point in the future.

{{< callout >}}

Always use the latest version of Ubuntu for the most up to date dependencies.

Note: Your mileage may vary, as GDM has some bugs with Hyprland. Check the [Master Tutorial](../Master-Tutorial) for more info.

{{< /callout >}}
```